### DISCUSSION TITLE
I have a new idea of a feature for blazor : Render website inside canvas

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/aspnetcore#48744] [Ideas]
https://github.com/dotnet/aspnetcore/issues/48705

Why not, pornhub would be very interested in this technology, since ad can now be uncensorable and a lot clients are suffering from ad block and you want them to be happy

**Comment by davidfowl** [comment]
https://github.com/dotnet/aspnetcore/issues/48705#issuecomment-1587693796

Seems like you got a reply.
